app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -357,12 +357,6 @@ export default {
       </div>
     </section>
 
-    <div v-if="clanIdSelected === ''" class="row flex-row text-center section-space free-to-get-started-image">
```

Comment:
Could we also please clean up CSS selector that no longer apply after deleting this code. Thanks!

https://github.com/codecombat/codecombat/blob/align_free_to_get_section_in_league/app/views/landing-pages/league/PageLeagueGlobal.vue#L801-L804